plasma_cash/child_chain/server.py

```diff
@@ -32,3 +38,22 @@ def submit_block():
 def send_tx():
     tx = request.form['tx']
     return container.get_child_chain().apply_transaction(tx)
+
+
+@ws.route('/')
+def websocket(socket):
+    global clients
```

Comment:
can we just put clients here?

Reply:
Considering the client notification may not always triggered from WebSocket (but from other regular API, eg. `send_tx`), I think the `clients` need to be a global variable.